Jenkinsfile.cloud

```diff
@@ -55,6 +55,21 @@ node(env.NODE) {
         """
     }
 
+    // do this again *after* running imgfac to close race
+    stage("Resync Repo") {
+        withCredentials([sshUserPrivateKey(credentialsId: env['ARTIFACT_SSH_CREDS_ID'],
+                                           keyFileVariable: 'KEY_FILE')]) {
+            sh """
+                mkdir -p ${output}/repo
+                rsync -Hrlpt --stats \
+                    -e 'ssh -i ${env.KEY_FILE} \
+                            -o UserKnownHostsFile=/dev/null \
+                            -o StrictHostKeyChecking=no' \
+                    ${env.ARTIFACT_SERVER}:${server_dir}/repo ${server_dir}
```

Comment:
I'm confused...aren't we pointing anaconda at the remote ostree-repo?  Why do we need to sync it locally?  What's `${output}/repo` versus `${server_dir}`?

Reply:
Now I understand this, although I think it'd be a lot less expensive to just do an `ostree pull --commit-metdata-only` or so later.